trompace/__init__.py

```diff
@@ -64,7 +66,10 @@ def make_parameters(**kwargs):
         elif isinstance(v, datetime.datetime):
             value = f"{{formatted: {encoder.encode(v.isoformat())}}}"
         elif isinstance(v, list):
+            #value = "[{}]".format(", ".join(item for item in encode_list(v, encoder)))
             value = "[{}]".format(", ".join(item for item in encode_list(v, encoder)))
+        elif isinstance(v, dict):
+            value = "{"+make_parameters(**v)+"}"
```

Comment:
A very small thing, but I'd prefer to have whitespace around the `+` in both cases here (and above in `encode_list`)